docs/manage/reference/accounts.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: "Account Management"
+title: "Account Management"
+weight: 1000
+layout: "docs"
+type: "docs"
+no_list: true
+description: "Log in and out of your Viam account."
+aliases:
+  - /fleet/account/
+  - /cloud/account/
+---
+
+The [Viam app](https://app.viam.com/) is a web UI for managing and building machines.
+
+## Create account and log In
+
+To get started on the Viam app, you must log in as an authorized user.
+Viam supports sign up using Google, GitHub, Apple, and email.
+
+Navigate to [the main page](https://app.viam.com/).
+If you haven't created an account yet, click **Sign Up** to create a new account using your preferred Single Sign On method or your email address and a password.
+If you already have an account, click **Log In** to log in using your Single Sign On credentials or your email address and password.
```

Comment:
```suggestion
If you haven't created an account yet, click **Sign Up** to create a new account using your preferred single sign on method or your email address and a password.
If you already have an account, click **Log In** to log in using your single sign on credentials or your email address and password.
```
[nit] It seems SSO is a more general concept rather than one company's product name